code/drasil-example/gamephysics/lib/Drasil/GamePhysics/References.hs

```diff
@@ -43,3 +43,15 @@ sciComp2013 = cArticle
   "PLoS Biol" 2013
   [volume 12, number 1] "sciComp2013"
 
+
+uriReferences :: [Reference]
+uriReferences = [accelGravitySrc, impulseSrc]
+
+accelGravitySrc :: Reference
+accelGravitySrc = makeURI "accelGravitySrc" "https://en.wikipedia.org/wiki/Gravitational_acceleration" $
+  shortname' $ S "Definition" `S.of_` S "Gravitational Acceleration"
+
+
+impulseSrc :: Reference
+impulseSrc = makeURI "impulseSrc" "http://www.chrishecker.com/images/e/e7/Gdmphys3.pdf" $
+  shortname' $ S "Impulse for Collision Ref"
```

Comment:
I moved the PEARs to the "References" file because I think that's most appropriate for where they belong. The PEARs should also contain authorship information, which is why they're next to the citations.